specifyweb/frontend/js_src/lib/components/WbPlanView/LineComponents.tsx

```diff
@@ -37,6 +37,7 @@ export type HtmlGeneratorFieldData = {
   readonly isDefault?: boolean;
   readonly isRelationship?: boolean;
   readonly tableName?: keyof Tables;
+  readonly tableTreeDefName?: string;
 };
```

Comment:
(off topic) haha, the name of this type `HtmlGeneratorFieldData`, is from the dark days before React when this was fed into a function that spit out HTML strings 🥲
we should probably rename it

Reply:
will create a new issue for it